styles/index.scss

```diff
@@ -59,13 +139,85 @@ a {
   }
 }
 
+ul {
+  text-align: center;
+
+  &:first-of-type {
+    @container body (min-width: 1024px) {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    @container body (min-width: 1440px) {
+      margin: 0;
+    }
+  }
+
+  /* li {
+    @container body (min-width: 1024px) {
+      padding-left: 52px;
+
+      &:first-of-type {
+        padding-left: 0;
+      }
+    }
+  } */
+
+  li > img {
+    padding: 50px 0 22px;
+
+    @container body (min-width: 1024px) {
+      padding: 88px 0 32px;
+    }
+  }
+
+  a {
+    text-decoration: underline;
+  }
```

Comment:
Since links globally across the designs have an underline, I think this might make more sense as a global style for `a` tags? The links in the nav that shouldn't have an underline already have a `text-decoration: none;` style on them so I don't think it should cause issues

Reply:
smart!